client/src/components/DrawingScreen/BrushConfig/BrushConfig.jsx

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+import React from 'react';
+import './BrushConfig.css';
+import PropTypes from 'prop-types';
+
+const Colors = Object.freeze({
+  // Map enum members to HTML colors
+  Black: 'black',
+  White: 'white',
+  Greenery: '#88B04B',
+  Coral: '#FF6F61',
+  Violet: '#6B5B95',
+  Marsala: '#955251',
+  Orchid: '#B565A7',
+  Turquoise: '#5B8AC',
+  Mimosa: '#EFC050',
+});
+
+const Widths = Object.freeze({
```

Comment:
nit: BrushSize here and throughout?

width.name is a bit hard to parse farther down

Reply:
makes sense!